Update Member UI to use MDL (dev: `yarn memberui`) & update reading list

XertroV opened the pull request:
- begin porting to Material Lite (via elm-mdl)
- add white text logo
- added elm-live
- add `memberui` npm script to work on member ui stuff - run with `yarn memberui`

XertroV committed:
Update Member UI to use MDL (dev: `yarn memberui`) & update reading list

- begin porting to Material Lite (via elm-mdl)
- add white text logo
- added elm-live
- add `memberui` npm script to work on member ui stuff - run with `yarn memberui`

ghost commented:
Your pull request doesn't follow our guidelines. Please fix the following:

* Pull request description must include a screenshot [(?)](https://gitmagic.io/rules/#/pull-request/body-must-include-screenshot)

[Click here](https://app.gitmagic.io/voteflux/vote-flux-v2/54/72c10fe5-5217-412a-b376-32a0e80ee5ee/ec55ed9a-a2ca-4efa-9143-8af2328eb100) for details.

Thank you! :pray:
<sub>
This comment was made by [GitMagic](https://gitmagic.io/) – Magically enforcing your contribution guidelines.
</sub>

XertroV committed:
Fix index.html

Note: I think if you run elm-make with no args it will overwrite index.html.
This is bad, but luckily I tested they deploy at least.

Something to keep in mind I suppose

XertroV committed:
Add slight change to test netlify status

XertroV commented:
Deploy preview ready!

Built with commit 3d1ea656843684dcbccebf63f416a77fec2262c5

https://deploy-preview-54--voteflux.netlify.com

XertroV committed:
Fix compiler bug
